public/browse.css

```diff
@@ -255,36 +255,63 @@ nav.pagination m-icon svg {
     0 0 0 3px var(--color-neutral-400) !important;
 }
 
+/*
+  Datastores navigation
+*/
+  
 .datastores-nav {
```

Comment:
All these changes look great! I'm glad to see the custom properties being useful.